webapp/portlet/src/main/webapp/vue-apps/space-menu/components/SpaceMenuItem.vue

```diff
@@ -39,26 +39,8 @@ export default {
       return this.navigation?.label;
     },
     icon() {
-      const navigationIcon = this.navigation && this.navigation.icon || '';
-      if (!navigationIcon?.length) {
-        return 'fas fa-sticky-note';
-      }
-      if (navigationIcon.includes('uiIconAppSpaceHomePage')) {
-        return 'fas fa-stream';
-      } else if (navigationIcon.includes('uiIconAppMembersPortlet')) {
-        return 'fas fa-users';
-      } else if (navigationIcon.includes('uiIconAppTasksManagement') || navigationIcon.includes('uiIconApptasks')) {
-        return 'fas fa-tasks';
-      } else if (navigationIcon.includes('uiIconAppNotes') || navigationIcon.includes('uiIconAppnotes') ) {
-        return 'fas fa-clipboard';
-      } else if (navigationIcon.includes('uiIconAppSpaceWallet')) {
-        return 'fas fa-wallet';
-      } else if (navigationIcon.includes('uiIconAppSpaceSettingPortlet')) {
-        return 'fas fa-cog';
-      } else {
-        return navigationIcon;
-      } 
-    },
+      return this.navigation?.icon || 'fas fa-folder';
```

Comment:
Shouldn't we use `icon-default-color icon-default-size` to be coherent with what was defined in [SpacePanelHamburgerNavigationItem.vue](https://github.com/Meeds-io/social/pull/3150/files?diff=unified&w=1#diff-2a0208ae61a39601be55f764e7e99b75d64fb1444dd44435472bc48baa0809c4) ?

```suggestion
      return `${this.navigation.icon || 'fas fa-folder'} icon-default-color icon-default-size`;
```

Reply:
fixed